### PR TITLE
Adds wemake-django-template

### PR DESCRIPTION
**Pull request by sobolevn**
List of people who use this template: https://github.com/wemake-services/wemake-django-template/wiki/Who-is-using-this-template

### For all PRs

- [x] I read the [Contribution Guide](CONTRIBUTING.md)

### For new stacks

- [x] I followed the [do's and don'ts](CONTRIBUTING.md#contributing-dos-and-donts)
- [x]  This stack does not already exist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,6 +300,24 @@ Self-hosted open source devops and collaboration tools, suitable for enterprise.
 - [Metabase](https://www.metabase.com/) - [🛠](https://stackshare.io/metabase) - [🐙](https://github.com/metabase/metabase) - An open-source business intelligence tool
 
 
+## Production Ready Django + Docker [↗](https://awesomestacks.dev/production-ready-django-docker)
+
+Bleeding edge django template focused on code quality and security.
+
+Build, test, and deploy pipelines are configured by default.
+
+- [Python3](https://www.python.org/) - [🛠](https://stackshare.io/python) - [🐙](https://github.com/cpython/python) - A programming language that lets you work quickly.
```
Comment (jdorfman) [inline README.md]: 1. Please change "Python3" to "Python 3"
1. Please fix the GitHub link to `https://github.com/python/cpython`

File: README.md [hunk 2]
```diff
@@ -300,6 +300,24 @@ Self-hosted open source devops and collaboration tools, suitable for enterprise.
 - [Metabase](https://www.metabase.com/) - [🛠](https://stackshare.io/metabase) - [🐙](https://github.com/metabase/metabase) - An open-source business intelligence tool
 
 
+## Production Ready Django + Docker [↗](https://awesomestacks.dev/production-ready-django-docker)
+
+Bleeding edge django template focused on code quality and security.
```
Comment (jdorfman) [inline README.md]: django = Django

**Comment by jdorfman** [comment]
@dzello any feedback?

**Comment by dzello** [comment]
@jdorfman LGTM 👍

**Pull request merged**